worlds/mm2/__init__.py

```diff
@@ -13,7 +13,7 @@
                         location_groups)
 from .rom import patch_rom, MM2ProcedurePatch, MM2LCHASH, PROTEUSHASH, MM2VCHASH, MM2NESHASH
 from .options import MM2Options, Consumables
-from .client import MegaMan2Client
+#from .client import MegaMan2Client
```

Comment:
```suggestion
from .client import MegaMan2Client
```
Should no longer be necessary following #4439

Reply:
I completely forgot I did that. Fixed.